juniper_official/System/pfe-traffic-statistics.rule

```diff
@@ -70,26 +90,26 @@ healthbot {
                 frequency 60s;
                 term are-host-path-drops-increasing {
                     when {
-                        min-rate-of-increase "$host-path-hardware-drops" { ## Warning: 'min-rate-of-increase' is deprecated
+                        min-rate-of-increase "$host-path-hardware-drop-count" { ## Warning: 'min-rate-of-increase' is deprecated
```

Comment:
Can we move away from min-rate-of-increase to incrasing-at-least-by?

Reply:
We will address through PR as change is required in many rules so we can do changes in all rule in one go.